test/functional/crossFunctional.mocha.js

```diff
@@ -774,6 +782,15 @@ describe("Cross Functional", function() {
         .limit(4)
     }));
 
+    it('works with length action on split', equalityTest({
+      executorNames: ['druid', 'mysql', 'postgres'],
+      expression: $('wiki').split({ 'PageLength': '$page.length()', 'Page': '$page' })
+        .sort('$PageLength', 'descending')
+        .apply('Count', '$wiki.sum($count)')
+        .sort('$Count', 'descending')
```

Comment:
There are two sorts here. Plywood will simplify them into one so might as well only have on in the test